addons/mail/models/mail_activity.py

```diff
@@ -77,12 +83,29 @@ def default_get(self, fields):
         ('today', 'Today'),
         ('planned', 'Planned')], 'State',
         compute='_compute_state')
+    next_activity_type_id = fields.Many2one("mail.activity.type", string="Recommended Activities")
```

Comment:
Actually you are right about that one. Could you rename it to recommanded_activity_type_id ?